### PR TITLE
chore: export extract element function

### PR DESCRIPTION
**Pull request by scomea**
# Description
Make extractHtmlElement function available for external use from the base components package.

## Motivation & context
Encountered a situation where it would be useful outside of the base components package.

## Issue type checklist
- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

**Comment by janechu** [comment]
Can you update the wording on this PRs description? It's unclear from the way it's written what you're trying to do.

File: packages/fast-components-react-base/src/index.ts [hunk 1]
```diff
@@ -114,7 +114,11 @@ import Typography from "./typography";
 export { Typography };
 export * from "./typography";
 
-export { WindowWithIntersectionObserver, WindowWithResizeObserver } from "./utilities";
+export {
+    WindowWithIntersectionObserver,
+    WindowWithResizeObserver,
+    extractHtmlElement,
```
Comment (nicholasrice) [inline packages/fast-components-react-base/src/index.ts]: We do have a `@microsoft/fast-react-utilities` package this may be better suited for.

Comment (scomea) [reply packages/fast-components-react-base/src/index.ts]: Oh yeah, I didn't want to put it in generic web utilities and missed that we had a react specific one.  I'll move it there.

Comment (scomea) [reply packages/fast-components-react-base/src/index.ts]: Moved, and also changed the ref prop to "any" as you noted in the other PR:
export function extractHtmlElement(
    sourceRef: React.RefObject<any> | HTMLElement
): HTMLElement | null {...

**Comment by scomea** [comment]
Re-setting reviews as I needed to add tests to make coverage and that revealed that some better handling of text nodes in VP was appropriate.

**Pull request merged**